nabsurprised/nabsurprised.py

```diff
@@ -61,7 +69,10 @@ async def _do_perform(self, expiration, lang, type):
         await self.writer.drain()
 
     def compute_random_delta(self, frequency):
-        return (256 - frequency) * 60 * (random.uniform(0, 255) + 64) / 128
+        key = frequency
+        if key not in NabSurprised.FREQUENCY_SECONDS:
+            key = NabSurprised.RARELY
+        return random.uniform(0, NabSurprised.FREQUENCY_SECONDS[key])
```

Comment:
This is my understanding of the original Violet (very obfuscated) Java code...
But shouldn't we rather have something like?:
```
if key == NabSurprised.VERY_OFTEN:
    return random.uniform(0, NabSurprised.FREQUENCY_SECONDS[VERY_OFTEN])
elif key == NabSurprised.OFTEN:
    return random.uniform(NabSurprised.FREQUENCY_SECONDS[VERY_OFTEN],NabSurprised.FREQUENCY_SECONDS[OFTEN])
elif key == NabSurprised.SOMETIMES:
    return random.uniform(NabSurprised.FREQUENCY_SECONDS[OFTEN],NabSurprised.FREQUENCY_SECONDS[SOMETIMES])
else:
    return random.uniform(NabSurprised.FREQUENCY_SECONDS[SOMETIMES],NabSurprised.FREQUENCY_SECONDS[RARELY])
```